### PR TITLE
Filter the dropdown options based on model availability in each region

### LINKED ISSUE #37: Disable options in dropdown
Typically, each domain is covered by only a subset of all CPM models. But if we select a domain, all groups are still shown in the dropdown. It would be nicer if the groups for which no CPM data is available would be disabled.

### PR DESCRIPTION
**Pull request by geek-yang**
Filter the dropdown options based on model availability in each region.

Closes #37

### CODE REVIEW

**Comment by Peter9192** [comment]
@geek-yang is this ready? If so, can you mark it ready and request my review?

**Comment by geek-yang** [comment]
Hi @Peter9192 , it is ready for review now. Apart from adding the filter feature, I also delete those maps not used in `static` folder. All maps are not stored in `static/cpm_analysis`.

File: components/cpm/Map.vue [hunk 1]
```diff
@@ -115,7 +115,8 @@ export default {
     },
     updateInfo (props) {
       // to do: loop through the json file after it is put in a geojson/yaml file
-      this._div.innerHTML = '<h4>CPM model&data availability</h4>' + (this.regions.regions[props] ? (this.regions.regions[props].model + '<br>' + this.regions.regions[props].SW + '<br>' + this.regions.regions[props].SE + '<br>' + this.regions.regions[props].NE + '<br>' + this.regions.regions[props].NW) : 'Hover over a region')
+      // this._div.innerHTML = '<h4>CPM model&data availability</h4>' + (this.regions.regions[props] ? (this.regions.regions[props].model + '<br>' + this.regions.regions[props].SW + '<br>' + this.regions.regions[props].SE + '<br>' + this.regions.regions[props].NE + '<br>' + this.regions.regions[props].NW) : 'Hover over a region')
```
Comment (Peter9192) [inline components/cpm/Map.vue:118]: Shall we remove the commented lines?

**Review by Peter9192 — APPROVED** [review]
This works nicely. Just a minor comment. 

I noticed that there are still some maps missing. I had expected that with the filtered dropdowns, maps would be available for all remaining options. Can we open a new issue about that?

**Comment by geek-yang** [comment]
> This works nicely. Just a minor comment.
> 
> I noticed that there are still some maps missing. I had expected that with the filtered dropdowns, maps would be available for all remaining options. Can we open a new issue about that?

Hi @Peter9192 , thanks for your comments. The missing maps are due to the file name. Some maps from the same group have different names in different regions. This is due to the original file name of Petter Lind's data. We only implement filtering but not handling naming exceptions. This will be fixed after we apply a uniformed naming convention in PR #50. I don't think it is necessary to open a new issue.

If after merging PR #50, we still got missing maps. Then we can open a new PR. Is that ok?

File: components/cpm/Map.vue (hunk 1 above)
Comment (geek-yang) [reply components/cpm/Map.vue:118]: Yes, I will clean it up!

**Pull request merged**